### PR TITLE
add more extensions

### PR DESCRIPTION
**Pull request by rntz**
Adds more extensions to the defaults in `file_extensions.py`.

### CODE REVIEW

**Review by phillco — APPROVED** [review]
It'd be nice to sort these. I guess we'd lose blame.

File: core/file_extension/file_extension.py [hunk 1]
```diff
@@ -32,6 +33,22 @@
...
     "dot text": ".txt",
+    "dot julia": ".jl",
+    "dot J L": ".jl",
+    "dot html": ".html",
+    "dot css": ".css",
+    "dot sass": ".sass",
+    "dot svg": ".svg",
+    "dot png": ".png",
+    "dot wave": ".wav",
+    "dot flack": ".flack",
```
Comment (auscompgeek) [inline core/file_extension/file_extension.py:44]: Was this mean to be `.flac`?

Comment (phillco) [reply core/file_extension/file_extension.py:44]: Whoops, missed that too :-(

Comment (phillco) [reply core/file_extension/file_extension.py:44]: https://github.com/knausj85/knausj_talon/pull/1076

**Pull request merged**